src/k8s-extension/HISTORY.rst

```diff
@@ -6,6 +6,7 @@ Release History
 1.6.5
 ++++++++++++++++++
 * microsoft.dataprotection.kubernetes: Add support for 'DisableInformerCache' configuration.
+* microsoft.azuremonitor.containers.metrics: simplify logic and enable correct recording rule groups for managed prom extension
```

Comment:
one nit: Start with upper case for simplify

Reply:
done